src/tests/conformance/post_handshake/validators.rs

```diff
@@ -257,14 +256,45 @@ async fn c026_TM_VALIDATOR_LIST_send_validator_list() {
         signature,
         version: 1,
     });
-    synth_node
+    synth_node1
         .unicast(node.addr(), payload)
         .expect("unable to send message");
 
-    // TODO: confirm result from rippled that the message was valid
-    // will be done in new PR
+    let check = |m: &BinaryMessage| {
+        if let Payload::TmValidatorListCollection(validator_list_collection) = &m.payload {
+            if let Some(blob_info) = validator_list_collection.blobs.first() {
+                let decoded_blob =
+                    base64::decode(&blob_info.blob).expect("unable to decode a blob");
+
+                let text = String::from_utf8(decoded_blob)
+                    .expect("unable to convert decoded blob to a string");
 
-    sleep(Duration::from_secs(5)).await;
-    synth_node.shut_down().await;
-    node.stop().expect("unable to stop stateful node");
+                let validator_list = serde_json::from_str::<ValidatorList>(&text)
+                    .expect("unable to deserialize a validator list");
+
+                // Only our message has a single validator, so we skip the others
+                if validator_list.validators.len() == 1 {
+                    assert!(validator_list.sequence == RAND_SEQUENCE_NUMBER);
+                    assert!(validator_list.validators[0]
+                        .validation_public_key
+                        .eq(MASTER_PUBLIC));
+                    return true;
+                }
+            }
+        }
+        false
+    };
+
+    timeout(TIMEOUT_MILLIS, async {
```

Comment:
Why not just `assert!(synth_node.expect_message(&check).await);` ? 
I know it's different timeout, but shouldn't make much difference here.